src/main/java/org/prebid/server/floors/PriceFloorFetcher.java

```diff
@@ -39,13 +39,18 @@
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class PriceFloorFetcher {
 
     private static final Logger logger = LoggerFactory.getLogger(PriceFloorFetcher.class);
 
     private static final int ACCOUNT_FETCH_TIMEOUT_MS = 5000;
     private static final int MAXIMUM_CACHE_SIZE = 300;
+    private static final int MIN_MAX_AGE_SEC_VALUE = 600;
+    private static final int MAX_AGE_SEC_VALUE = Integer.MAX_VALUE;
+    private static final Pattern CACHE_CONTROL_HEADER_PATTERN = Pattern.compile("^.*(max-age=\\d+).*$");
```

Comment:
You're only interested in the value of the `max-age`. Move your `/d+` matcher to the capture group and access it directly. You won't need to split your result then.

Reply:
Nice catch, thx)